python/ray/train/torch/torch_checkpoint.py

```diff
@@ -86,6 +88,8 @@ def from_state_dict(
         Args:
             state_dict: The model state dictionary to store in the checkpoint.
             preprocessor: A fitted preprocessor to be applied before inference.
+            training_state: A dictionary that contains the objects you need to resume
+                training.
```

Comment:
This is not very clear to me, can we describe this as something like "additional training state to checkpoint. This can be accessed by `checkpoint.get_training_state()`." or so